styles/blocks/form.scss

```diff
@@ -0,0 +1,76 @@
+@use '../utils/mixins' as *;
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  &__input {
+    background: #f8f8fa;
+    border-radius: 40px;
+    border: 1px solid #f8f8fa;
+    width: 100%;
+    height: 48px;
+    line-height: 48px;
+    padding-left: 17px;
+    box-sizing: border-box;
+
+    &::placeholder {
+      font-family: 'Inter';
+      font-style: normal;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 20px;
+      color: #7E7E83;
+    }
+
+    &:hover {
+      border: 1px solid #D5D7DE;
+      box-sizing: border-box;
+
+      &::placeholder {
+        color: $main-font-color;
+      }
+    }
+
+    &:focus {
+      outline: 1px solid $main-font-color;
+    }
+  }
+
+  &__textarea {
+    background: #f8f8fa;
```

Comment:
Avoid using direct color codes. Use SCSS variables for colors to maintain consistency and reusability.